queue.tf

```diff
@@ -0,0 +1,30 @@
+module "queue-namespace" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-namespace?ref=master"
+  name                = "${local.bsp_product}-servicebus-${var.env}"
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.bulkscan_rg.name}"
+  env                 = "${var.env}"
+  common_tags         = "${var.common_tags}"
+}
+
+module "notifications-queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=master"
+  name                = "notifications-reform"
+  namespace_name      = "${module.queue-namespace.name}"
+  resource_group_name = "${azurerm_resource_group.bulkscan_rg.name}"
+  lock_duration       = "PT5M"
+
+  duplicate_detection_history_time_window = "PT15M"
```

Comment:
needs tf formatting :)

Reply:
all good here. the parameter is too long and kept separate from the previous block. otherwise formatting will look ridiculously indented :)